<?php

namespace Sigae\Config;
use Exception;
use PDO;
use PDOException;

function conectarDB($user, $pswd, $hostname){

    try {
        // Creacion de conexi贸n a la base de datos
        $conexionBD = new PDO("mysql:host=$hostname:3306;dbname=aio_db", $user, $pswd);
        $conexionBD->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION); 
        return $conexionBD;
    } catch(PDOException $e) {
        // Retornar un mensaje de error, lanza una excepci贸n
        error_log("Error de conexi贸n: " . $e->getMessage());
        throw new Exception("Fallo en la conexi贸n a la base de datos: " . $e->getMessage());
    }
    exit();
}

?>
